lighthouse-core/scripts/lantern/collect/gcp-run.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+whoami
+export HOME="/home/lighthouse"
+
+cd /home/lighthouse
+mkdir -p ./src
+cd ./src
+
+if [[ ! -d ./lighthouse ]]; then
+  git clone https://github.com/GoogleChrome/lighthouse.git
```

Comment:
the `create` gcp command above means this script always runs in a new instance, right? (wondering if we need to git pull)

Reply:
yeah but if something goes wonky and you need to restart it's nice to not have to manually vi scripts if you're talking about removing it.

if you're talking about adding it, git pull is effectively done below